src/components/Workshop/RecyclingTabPanel.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import Divider from '@material-ui/core/Divider'
+import ReactMarkdown from 'react-markdown'
+
+import { recipeType } from '../../enums'
+
+import { recipeCategories } from '../../data/maps'
+
+import { TabPanel } from './TabPanel'
+import { RecipeList } from './RecipeList'
+
+export function RecyclingTabPanel({
```

Comment:
Nice! 😁

Reply:
this was so easy thanks to the panel refactors done recently. very glad i did that 😄